dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/get-fee-record-correction-review.controller/helpers.test.ts

```diff
@@ -0,0 +1,237 @@
+import {
+  CURRENCY,
+  FeeRecordCorrectionEntityMockBuilder,
+  FeeRecordCorrectionReviewInformation,
+  FeeRecordEntityMockBuilder,
+  getFormattedMonetaryValue,
+  RECORD_CORRECTION_REASON,
+  RecordCorrectionReason,
+  UtilisationReportEntityMockBuilder,
+} from '@ukef/dtfs2-common';
+import {
+  getFormattedFormDataValueForCorrectionReason,
+  mapFormDataToFormattedValues,
+  mapTransientCorrectionDataToReviewInformation,
+  validateRequiredFormDataField,
+} from './helpers';
+import { mapCorrectionReasonsToFormattedOldValues } from '../../../../../helpers/map-correction-reasons-to-formatted-values';
+
+console.error = jest.fn();
+
+describe('get-fee-record-correction-review.controller helpers', () => {
+  describe('validateRequiredFormDataField', () => {
+    it('should not throw error when value is defined and not null', () => {
+      // Arrange
+      const value = 'test';
+      const reason = RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT;
+
+      // Act & Assert
+      expect(() => validateRequiredFormDataField(value, reason)).not.toThrow();
+    });
+
+    it('should not throw error when value is zero', () => {
+      // Arrange
+      const value = 0;
+      const reason = RECORD_CORRECTION_REASON.REPORTED_FEE_INCORRECT;
+
+      // Act & Assert
+      expect(() => validateRequiredFormDataField(value, reason)).not.toThrow();
+    });
+
+    it('should throw error when value is undefined', () => {
+      // Arrange
+      const value = undefined;
+      const reason = RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT;
+      const expectedError = `Required field is missing from transient form data for correction reason: ${reason}`;
+
+      // Act & Assert
+      expect(() => validateRequiredFormDataField(value, reason)).toThrow(expectedError);
+    });
+
+    it('should throw error when value is null', () => {
+      // Arrange
+      const value = null;
+      const reason = RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT;
+      const expectedError = `Required field is missing from transient form data for correction reason: ${reason}`;
+
+      // Act & Assert
+      expect(() => validateRequiredFormDataField(value, reason)).toThrow(expectedError);
+    });
+  });
+
+  describe('getFormattedFormDataValueForCorrectionReason', () => {
+    it(`should map form data value for reason "${RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT}" to original value`, () => {
```

Comment:
I got slightly confused about original value here because this helper is for the new values and it took me a moment to understand it just meant should return the form value as is. What do you think about:
```suggestion
    it(`should return the form data facilityId value for reason "${RECORD_CORRECTION_REASON.FACILITY_ID_INCORRECT}" `, () => {
```
instead?